src/feature/relay/selftest.h

```diff
@@ -15,24 +15,29 @@
 #ifdef HAVE_MODULE_RELAY
 
 struct or_options_t;
-int router_should_skip_orport_reachability_check(
-                                         const struct or_options_t *options);
+#define router_should_skip_orport_reachability_check(opts) \
+  router_should_skip_orport_reachability_check_family((opts),0)
+int router_should_skip_orport_reachability_check_family(
+                                         const struct or_options_t *options,
+                                         int family);
```

Comment:
Hmm, is the `_family` really necessary in the name? I'm asking because some other functions take a `family` argument but do not have it in their name but this one does for some reasons.

For instance `router_do_orport_reachability_checks()` or `router_orport_found_reachable()`. I don't have a strong opinion here although I would lean towards **not** having the "family" in the name but either way, I would just make it the same standard for everyone regardless of what you decide.

Reply:
My reason for using a special name for this function was that there is already a `router_should_skip_orport_reachability_checks()` that checks for _all_ families.  

Honestly I think we should rename all of these functions completely -- 'should skip' seems wrong here.  Okay if I open a new ticket for this?

Reply:
Sounds good!